src/components/ui/avatar/Avatar.mdx

```diff
@@ -0,0 +1,114 @@
+# Avatar
+
+import { Controls, Canvas, Meta, Story, Source } from '@storybook/blocks';
+import * as AvatarStories from './Avatar.stories';
+
+<Meta of={AvatarStories} />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Examples](#examples)
+
+## Overview
+
+A graphical representation of the user, often used in profile sections.
```

Comment:
An avatar is a visual representation of a user, usually in the form of a profile picture, initials, or an icon that represents the user. It serves as a visual identifier for users, making it easier to distinguish between different individuals in a digital environment.